### ISSUE TITLE
Will #random=1# cfg increase mAP in the end?

### ISSUE THREAD

**Issue by MingPap** [AlexeyAB/darknet#2080]

**Comment by AlexeyAB** [comment]
If `random=1` then

* On validation dataset mAP will be higher.
* On training dataset mAP will be lower.

So it is better to use `random=1`

**Comment by MingPap** [comment]
@AlexeyAB  Thanks

**Comment by tamim662** [comment]
> If `random=1` then
> 
> * On validation dataset mAP will be higher.
> * On training dataset mAP will be lower.
> 
> So it is better to use `random=1`

@AlexeyAB I have also seen that set random to 1 will affect to get better mAP on the validation set, but can you give the idea of how random parameter works to get better mAP in the validation set YOLO model?
